src/app/entities/entitiesList/CreateNewEntity.tsx

```diff
@@ -0,0 +1,25 @@
+'use client';
+import { addGameEntity } from '@/app/entities/entitiesActions';
+import { toast } from 'react-toastify';
+import React from 'react';
+
+export const CreateNewEntity = () => {
+  const addNewEntity = async (form: FormData) => {
+    try {
+      await addGameEntity(form);
+    } catch (error: any) {
+      toast(error.message);
```

Comment:
shouldn't we use error handling from next instead?

Reply:
we're still testing error boundaries & didn't manage to make it work at the time, so this is just a poc of a feedback system. I suggest we merge this and rethink how we do errors using e.g. [this tweet](https://twitter.com/dan_abramov/status/1654523681639366657) @Mati20041 linked elsewhere as inspo